src/data/finsupp/basic.lean

```diff
@@ -1662,6 +1662,28 @@ protected def dom_congr [add_comm_monoid β] (e : α₁ ≃ α₂) : (α₁ →
   end,
   map_add' := λ a b, map_domain_add, }
 
+@[to_additive]
+lemma mul_equiv.map_finsupp_prod {α β γ δ : Type*}
```

Comment:
Two comments:

* Is it inside `finsupp` namespace? It shouldn't.
* Please indent arguments with two spaces.

Reply:
Good catch, fixed